WultraMobileTokenSDKTests/NetworkingObjectsTests.swift

```diff
@@ -88,6 +88,7 @@ class NetworkingObjectsTests: XCTestCase {
                 XCTAssert(attr.label.value == "Částka")
                 XCTAssert(attr.amount ==  NSDecimalNumber(string: "965165234082.23").decimalValue)
                 XCTAssert(attr.currency == "CZK")
+                XCTAssert(attr.valueFormatted == "\(NSDecimalNumber(string: "965165234082.23").decimalValue) CZK")
```

Comment:
`NSDecimalNumber` is not necessary to use here. In the case of an amount, we're testing decimal parsing and this is not the case. Just use `"965165234082.23 CZK"`.